lib/generators/stimulus_reflex/templates/config/importmap.rb.tt

```diff
@@ -0,0 +1,17 @@
+# Pin npm packages by running ./bin/importmap
+
+pin "application", preload: true
+pin "@hotwired/turbo-rails", to: "turbo.min.js"
+pin "@hotwired/stimulus", to: "stimulus.min.js"
+pin "@hotwired/stimulus-loading", to: "stimulus-loading.js"
+
+pin "@rails/actioncable", to: "actioncable.esm.js"
+pin "cable_ready", to: "https://devbuilds.herokuapp.com/package/npm/cable_ready/latest"
+pin "stimulus_reflex", to: "https://devbuilds.herokuapp.com/package/npm/stimulus_reflex/latest"
```

Comment:
We should default these URLs to the version the Gem ships, otherwise we have version mis-matches again.
```suggestion
pin "cable_ready", to: "cable_ready.min.js"
pin "stimulus_reflex", to: "stimulus_reflex.min.js"
```

Reply:
As we've established, I'm not even remotely expert with importmap, so have some patience...

I've only got links to your heroku project for the duration of the development - basically while we work on this PR. My assumption is that we'd swap out right before cutting a non-preview release. Does that make sense?

Reply:
Realizing that usng your `devbuilds` service with `latest` means that if there's multiple PRs in active development, you are fighting over which branch has "control" of `latest`.

Can I suggest that you add the ability to access the latest build for a particular branch? Something like:
```ruby
pin "cable_ready", to: "https://devbuilds.herokuapp.com/package/npm/cable_ready/latest/new_installer"
# or
pin "cable_ready", to: "https://devbuilds.herokuapp.com/package/npm/cable_ready/new_installer/latest"
```